frontend/src/state/reducers/advisorReducer.ts

```diff
@@ -0,0 +1,73 @@
+import produce from "immer";
+import { getType } from "typesafe-actions";
+import { AdvisorAction } from "../actions";
+import {
+  setEmail,
+  setImage,
+  setName,
+  setStudents,
+  setToken,
+  setUserId,
+} from "../actions/advisorActions";
+
+export interface Student {
+  readonly email: string;
+  readonly name: string;
+}
```

Comment:
This simply contains information needed to display a list of users to the advisor, where the student's email uniquely defines them. 

The idea is that when an advisor selects a student, this email is used to fire off a new request to retrieve the rest of that student's information.